src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -114,7 +128,12 @@ public boolean isLargeDiscrepancyBetweenMegaTag1And2(int limelightNumber) {
    *     return 0 for x, y, and theta
    */
   public PoseEstimate getMegaTag1PoseEstimate(int limelightNumber) {
-    return LimelightHelpers.getBotPoseEstimate_wpiBlue(getLimelightName(limelightNumber));
+    try {
+      return LimelightHelpers.getBotPoseEstimate_wpiBlue(getLimelightName(limelightNumber));
+    } catch (Exception e) {
+        System.err.println("Error estimating MegaTag1 pose for the: " + getLimelightName(limelightNumber) + e.getMessage());
```

Comment:
Can LimelightHelpers.getBotPose... actually throw any exceptions? If not, we should consider moving this try catch somewhere further up in the vision subsystem.

Reply:
Dang I think u right

Reply:
Maybe we put them surrounding each of the threads?

Reply:
there is already a try catch in the visionThread and the checkAndUpdatePose. so we might be able to just remove this one

Reply:
yeah